examples/iotivity_examples/README.md

```diff
@@ -0,0 +1,314 @@
+```
+We will use Serial Numbers in order to identify the designed node during the compilation phase.
+
+###<a name="l2n_br"></a>Start the Border Router
+Step 1) Open a terminal window in `/example/iotivity-examples/br_fw/` and type
+```
+$ make flash BOARD=samr21-xpro SERIAL=br_node_serial
+```
+Step 2) Once the flashing is finished, we have to open a network interface. Type
+```
+$  PORT=/dev/ttyACM0 make term
+```
+Step 3) We have to complete the routing strategy on the BR, so in the RIOT shell type
+```
+> ifconfig 7 add 2001:db8::2
+> fibroute add :: via 2001:db8::1 dev 7
+```
+Now the BR is ready. The network is configured as follow:
+- Serial Port: **/dev/ttyACM0**
+- Interface: **tap0**
+- Address Prefix: **2001:db8::\64**
+- Routing on tap0 of multicast packet with destination **ff03::158**
+
+It is possible to configure the network with different parameters by invoking directly the initialization script instead of executing the Step 2:
+```
+$ make host-tools
+$ sudo ./start_network_mcast.sh <serial-port> <tap-device> <IPv6-prefix> <IPv6-mcast>
+```
+then run the Step 3 with the proper changes.
+
+###<a name="l2n_srv"></a>Start the Server
+Open a terminal window, go to `/examples/iotivity-examples/server` and type
+```
+$ make flash BOARD=samr21-xpro SERIAL=server_node_serial
+```
+then we open the serial connection
+```
+$ make term BOARD=samr21-xpro SERIAL=server_node_serial
+```
+The server starts the initialization phase, then it is ready for incoming requests. We can check the reachability of the server by typing in another terminal window
+```
+$ ping6 <IPv6 server>
+```
+
+###<a name="l2n_out"></a>Server Output
+Managing a GET request the output is like
+```
+2016-11-04 14:39:32,115 - INFO # ipadapter: got server request
+2016-11-04 14:39:32,121 - INFO # Incoming message from [2001:0db8:0000:0000:0000:0000:0000:0001]:56214
+2016-11-04 14:39:32,125 - INFO # ipadapter: waiting for server requests...
+2016-11-04 14:39:32,128 - INFO # server_oic: GET request
+2016-11-04 14:39:32,130 - INFO # server_oic: Light state 0
+2016-11-04 14:39:32,145 - INFO # Outgoing message to [2001:0db8:0000:0000:0000:0000:0000:0001]:56214
+```
+Managing a PUT request the output is like
+```
+2016-11-04 14:39:35,119 - INFO # ipadapter: got server request
+2016-11-04 14:39:35,125 - INFO # Incoming message from [2001:0db8:0000:0000:0000:0000:0000:0001]:56214
+2016-11-04 14:39:35,128 - INFO # ipadapter: waiting for server requests...
+2016-11-04 14:39:35,134 - INFO # server_oic: PUT request
+2016-11-04 14:39:35,136 - INFO # server_oic: key: state value: 1
+2016-11-04 14:39:35,138 - INFO # server_oic: LED0 is ON
+2016-11-04 14:39:35,141 - INFO # Outgoing message to [2001:0db8:0000:0000:0000:0000:0000:0001]:56214
+```
+
+###<a name="l2n_tst"></a>Testing
+There are many different ways to test this scenario.
+
+ - Tools: you can use [Copper(Cu)][7] or [coap-cli][8] to perform get request. The second one supports CBOR.
```

Comment:
is CBOR needed to make this example work? If yes and if Cu does not support CBOR, I propose you do not mention Cu? Else, why mention CBOR at this point?